index.md

```diff
@@ -52,8 +52,29 @@ because your past self doesn't answer email.
 
 ## Data Management
 
+Your project data may need to exist in various forms, ranging from raw to highly processed. Here are some guiding principles to help you move data through this developmental process:
+
+**First, do no harm.** Save data in the rawest form available and resist the temptation to overwrite these files with cleaner versions. This could be the data file produced by an instrument or raw results from a survey, with all of their mystifying imperfections. Faithful retention guarantees you can re-run your analysis, nachos to cheesecake, in the future. Long-term, this level of reproducibility enhances everyone's confidence in your final results. The more immediate payoff is the ability to recover gracefully from analytical mishaps and the freedom to experiment without fear.
+
+**Be the raw data you wish to see in the world.** From the raw data, make the dataset you *wish* you had received and that you will enjoy having as the start point for downstream analyses, some of which you have not even dreamed of yet. This is the place to maximize machine and human readability, preferably at the same time. It is neither the place to do vigorous data filtering nor to the time to bring in external information. Approach this initial preparation as internal to the existing dataset and non-destructive, both of the data and its general "shape". Enhance machine readability by converting from proprietary and high-friction formats (e.g., Microsoft Excel or XML) to open and simple formats (e.g., comma delimited plain text). Enhance human readability by replacing inscrutable variable names and artificial data codes with self-explaining alternatives. For example, rename the variable SOME_AWFUL_GENERIC_NAME to SOMETHING_DESCRIPTIVE *(JB: anyone ... got a good example that's real/realistic but will resonate with many?)*, recode the treatment variable from `1` vs. `2` to `untreated` vs. `treated`, and replace artificial codes, such as "-99" for missing data, with proper `NA`s. Both human and machine readability can be enhanced storing especially useful metadata as part of the filename itself, while keeping the filename regular-expression-friendly.
```

Comment:
XML is very machine-readable. Just not human-readable...

SOME_AWFUL_GENERIC_NAME --> name1, name2 to firstName, lastName

can be enhanced storing especially --> can be enhanced by storing especially 

Not  sure all readers will be familiar with regular expressions...